desktop/src/views/Pro/Workspace/Logs.tsx

```diff
@@ -23,10 +27,11 @@ import {
   VStack,
 } from "@chakra-ui/react"
 import dayjs from "dayjs"
-import { useEffect, useState } from "react"
+import { JSXElementConstructor, ReactElement, useEffect, useMemo, useRef, useState } from "react"
```

Comment:
couple of leftover imports it seems like?

Reply:
Done